src/RobotSimulator/StdWorldBuilder.ts

```diff
@@ -75,6 +75,20 @@ export class StdWorldBuilder {
 
     const robotSpec = robotBuilder.generateSpec();
     robotSpec.initialPosition = this.startPosition;
+    robotSpec.customMesh = {
+      filePath: `${process.env.PUBLIC_URL}/assets/models/robot_full_compressed.glb`,
```

Comment:
`process.env.PUBLIC_URL` how does that work? there are no environment variables in the web browser?

Reply:
https://create-react-app.dev/docs/using-the-public-folder/ - react essentially replaces it with the public folder location